LocalSettings.php

```diff
@@ -426,8 +426,7 @@
 	],
 	//CommonsMetadata
 	'wgCommonsMetadataForceRecalculate' => [
-		'default' => false,
-		'test1wiki' => true,
+		'default' => true,
```

Comment:
Why is this being enabled by default?

Reply:
As it allows remote files (from miraheze/wikimedia) commons to be used if enabled for this extension. As far as I can tell, those images wouldn't work without it.

Reply:
This should be made configurable in MW and disabled by default.

Based on me reading https://www.mediawiki.org/wiki/Extension:CommonsMetadata/en there may be a performance hit with this enabled. Secondly this you can already use remote files.

What the extension does is it pulls meta data.